reframe/core/modules.py

```diff
@@ -207,6 +208,30 @@ def is_module_loaded(self, name):
     def _is_module_loaded(self, name):
         return self._backend.is_module_loaded(Module(name))
 
+    def load_mapping_from_file(self, module_mapping_file):
+        """Update ``self.module_map`` with a mapping defined in a file."""
```

Comment:
* Do not use `self.module_map` in the docstring. Better write "Update the internal module mapping [...]".
* `module_mapping_file` -> `filename`.